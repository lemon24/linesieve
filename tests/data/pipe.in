-s '(\S+):$'
pipe -s one 'cut -d- -f1'
pipe -s two 'echo yyy'
pipe -s true true
pipe -s false false
---
one:
a-b
c-d
true:
t
false:
f
two:
xxx
xxx
