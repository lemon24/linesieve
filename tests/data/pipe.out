one
a
c
true
false
linesieve pipe: false exited with status code 1
two
yyy
